### ISSUE TITLE
Tables Refactoring Analysis and POC

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#7312]
In order to improve **Accessibility Compliance** and **Design System Consistency**, a tables refactoring strategy is needed.

- Why we need to refactor:
https://wiki.wdf.sap.corp/wiki/display/spar/Tables+Refactoring
- Tables POC Conclusions **(follow the link to review POC code)**:
https://wiki.wdf.sap.corp/wiki/display/spar/Tables+POC+Conclusions

**Comment by developpeurweb** [comment]
Standalone Tables POC needed for Screen Reader testing and determine the best hearing experience.
E.g.

- [x] Only Real Tables
- [x] Lists styled as tables
- [x] Tables containing list.

Beware of known WCAG failures:
https://www.w3.org/WAI/WCAG21/Techniques/failures/F91

**Comment by KateChuen** [comment]
Tested locally on develop branch.
- Order History
AMP violation on returns tab https://jira.tools.sap/browse/CXSPA-450


<img width="930" alt="orderHistoryEntering" src="https://user-images.githubusercontent.com/45409515/170099360-425997df-0b74-45bb-81de-0e98f2247051.png">

![orderHistoryLeaving](https://user-images.githubusercontent.com/45409515/170099357-9bae6cb4-09f2-46ec-ae8a-53e5bb23651e.png)

<img width="872" alt="Screen Shot 2022-05-24 at 1 45 49 PM" src="https://user-images.githubusercontent.com/45409515/170099462-0c3dac04-b068-4c4c-a41c-e5598ebc2dce.png">


<img width="1151" alt="Screen Shot 2022-05-24 at 1 53 55 PM" src="https://user-images.githubusercontent.com/45409515/170101512-cbc6ea50-40c1-49d9-8979-d985755495a0.png">

<img width="1224" alt="Screen Shot 2022-05-24 at 1 54 17 PM" src="https://user-images.githubusercontent.com/45409515/170102095-f659a042-c2c2-434d-b5ad-6d43ccae7e61.png">


- Cart page OK

![cartEntering](https://user-images.githubusercontent.com/45409515/170098407-545e489c-8735-4493-9c95-0c37961f1ca2.png)

![cartLeaving](https://user-images.githubusercontent.com/45409515/170098413-f9e36c9a-4f9b-4e5c-8021-28ee6ee5b73e.png)

![saveForLaterEntering](https://user-images.githubusercontent.com/45409515/170098417-b9db9c9d-5df1-4514-8d5f-1050e84b85c2.png)

![saveForLaterLeaving](https://user-images.githubusercontent.com/45409515/170098420-9f65e83d-1bf9-49ec-a2a1-db027530c0ba.png)

<img width="870" alt="Screen Shot 2022-05-24 at 1 42 30 PM" src="https://user-images.githubusercontent.com/45409515/170098817-dac30d36-321a-4dc2-8035-e01a1b600484.png">


- checkout review - OK

![OrderReviewEntering](https://user-images.githubusercontent.com/45409515/170097598-3b7df1c0-b49f-48f2-9d0a-807baeea3cef.png)

![OrderReviewLeaving](https://user-images.githubusercontent.com/45409515/170097606-f6a3e3a3-d253-4725-85eb-7cf4adf4693f.png)

<img width="875" alt="Screen Shot 2022-05-24 at 1 35 55 PM" src="https://user-images.githubusercontent.com/45409515/170097724-fdca9ea7-7286-4f0d-a216-7c4d3e6743e2.png">


- order confirmation - OK
![OrderConfirmationEntering](https://user-images.githubusercontent.com/45409515/170097348-e2fc3acd-93bb-45bc-883d-91b33aafc163.png)

![orderConfirmationLeaving](https://user-images.githubusercontent.com/45409515/170097352-f96e891c-3af9-4b87-9b7c-8633549eccaf.png)

<img width="849" alt="Screen Shot 2022-05-24 at 1 32 07 PM" src="https://user-images.githubusercontent.com/45409515/170097108-6cd4c3d7-cff4-44b1-a3db-b591de440f2b.png">

- cancel returns

![cancelOrderEntering](https://user-images.githubusercontent.com/45409515/170099331-8f006712-e9a1-4c9c-aa07-fd01dfea1b38.png)

![cancelOrderLeaving](https://user-images.githubusercontent.com/45409515/170099327-c13714d8-a25d-497d-909e-ff60417cdd41.png)

<img width="843" alt="Screen Shot 2022-05-24 at 1 47 30 PM" src="https://user-images.githubusercontent.com/45409515/170099805-60057a40-6ea9-4672-b853-7d5105fb2f26.png">


<img width="1191" alt="Screen Shot 2022-05-24 at 1 52 59 PM" src="https://user-images.githubusercontent.com/45409515/170101643-9cbaa3f4-53d2-4484-9c34-e4ec4f4d4aff.png">


<img width="1174" alt="Screen Shot 2022-05-24 at 1 53 29 PM" src="https://user-images.githubusercontent.com/45409515/170101587-e16335b3-5a21-47da-ba41-c4bb35e77e88.png">

<img width="848" alt="Screen Shot 2022-05-24 at 1 56 55 PM" src="https://user-images.githubusercontent.com/45409515/170101413-ed299a9a-2636-4ea3-8004-93a6af6af98e.png">

- wish list OK
![wishListEntering](https://user-images.githubusercontent.com/45409515/170102624-65da5c0c-229b-4b94-b5ad-d0c39cf041ce.png)

![wishListLeaving](https://user-images.githubusercontent.com/45409515/170102634-65a41b9e-bbc4-4a89-929e-ff9e04d85b6b.png)

<img width="843" alt="Screen Shot 2022-05-24 at 2 03 41 PM" src="https://user-images.githubusercontent.com/45409515/170102663-57c0feaa-ff7f-44a4-8ce0-95ca9522095f.png">

- my interests

AMP violations: https://jira.tools.sap/browse/CXSPA-499

![MyInterestsEntering](https://user-images.githubusercontent.com/45409515/170103926-662f82ac-eb02-4fc7-81c2-b80092b461bd.png)

![MyInterestsLeaving](https://user-images.githubusercontent.com/45409515/170103922-6e515bb4-6be0-4f78-8f8c-390d3aa240b8.png)
